src/silverlabnwb/nwb_file.py

```diff
@@ -660,15 +674,14 @@ def read_functional_data(self, folder_path):
         self.log('Loading functional data from {}', folder_path)
         assert os.path.isdir(folder_path)
         # Figure out timestamps, measured in seconds
-        epoch_names = self.nwb_file.epochs[:, 'epoch_name']
-        trials = [int(s[6:]) for s in epoch_names]  # names start with 'trial_'
+        trials = [s+1 for s in self.nwb_file.trials.id.data]
```

Comment:
As far as I can see, we only ever care about the number of trials/epochs here, so there's a chance to simplify this a bit!